sdk/containerregistry/azure-containers-containerregistry/README.md

```diff
@@ -56,7 +56,7 @@ and then include the direct dependency in the dependencies section without the v
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-containers-containerregistry</artifactId>
-  <version>1.1.2</version>
+  <version>1.2.0</version>
```

Comment:
surprisingly see this `current` version tag to be updated to a `dependency` tag value. 
But will revert the change anyway.